extensions/adobe/experience/adcloud/dsp/campaign.schema.json

```diff
@@ -0,0 +1,107 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/dsp/campaign",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "DSP Advertising Campaign",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Adobe Advertising Cloud DSP Campaign Details.",
+  "definitions": {
+    "dsp-campaign": {
+      "properties": {
+        "xdm:campaignKey": {
+          "title": "Campaign Key",
+          "type": "string",
+          "description": "Campaign external identifier."
+        },
+        "xdm:campaignName": {
+          "title": "Campaign Name",
+          "type": "string",
+          "description": "Name of the campaign."
+        },
+        "xdm:campaignStatus": {
+          "title": "Campaign Status",
+          "type": "string",
+          "description": "Campaign Status extracted from the Insertion Order associated with this campaign.Indicates if underlying ads are eligible to serve.",
+          "enum": [
+            "Active",
+            "Inactive",
+            "Deleted",
+            "Paid"
+          ],
+          "meta:enum": {
+            "Active": "Active",
+            "Inactive": "Inactive",
+            "Deleted": "Deleted",
+            "Paid": "Paid"
+          }
+        },
+        "xdm:timezone": {
+          "title": "Campaign Timezone",
+          "type": "string",
+          "description": "Timezone of the campaign."
+        },
+        "xdm:type": {
+          "title": "Campaign Type",
+          "type": "string",
+          "description": "Campaign Type",
+          "enum": [
+            "Media",
+            "Brandsights"
+          ],
+          "meta:enum": {
+            "Media": "Media",
+            "Brandsights": "Brandsights"
+          }
+        },
+        "xdm:userId": {
+          "title": "User Identifier",
+          "type": "integer",
```

Comment:
Rest of the identifiers are defined as of type "string". We should be OK with integer but I would like to flag this. Evaluate whether "string" data type makes sense for userId.

Reply:
userId is our internal DB identifier, not mapped to any record:id so ,I assume, no need to convert it to string.